tests/unit/menu/helper.js

```diff
@@ -14,7 +14,7 @@ return $.extend( helper, {
 		if ( message === undefined ) {
 			message = lastItem;
 		}
-		log.push( $.trim( message ) );
+		log.push( String.prototype.trim.call( message ) );
```

Comment:
In my mind, this calls into question the replacement of `$.trim(…)` with `….trim()` in source files. Since the PR isn't intended to make any changes other than restoring functionality, perhaps we should be doing `String.prototype.trim.call(…)` everywhere (or defining a local `trim` function and using that, like you're doing with `isWindow`).

Reply:
@gibson042 I changed the order of commits to make the `$.trim` changes the last one and modified all replacements to use `String.prototype.trim.call` instead, as you advised.

While an util would be easier to use, I don't see any infrastructure in UI to create internal utility modules that can be used in other files and this change is needed in various parts of the repository. Not wanting to expose anything new on `$.ui` (even if "internal" in theory), I went for `String.prototype.trim.call`.